src/services/container/ContainerService.js

```diff
@@ -18,134 +19,179 @@ export class ContainerService extends BaseService{
     /**
      * Creates a new container and returns the container object with an id
      * @param {string} name name of the new container
-     * @param {string} householdId id of the household where the container is held
      * @returns the container object that was created
+     * @throws errors if a container was not able to be made in DB
      */
-    async createContainer(name, householdId, icon = this.defaultIcon){
-        this.__UseCollection(this.CONTAINER_COLLECTION)
-        
-        
-        const container = new Container(name, householdId, [], icon)
-
-        const newDocId = await this.__CreateEntity( container.toDocument() )
-        container.id = newDocId
-        return container
+    async createContainer(name){
```

Comment:
Please pull most recent changes for master, I have the default icon be the default argument so that it would not be a breaking change.